src/tests/Title.test.js

```diff
@@ -3,8 +3,12 @@ import { mount } from 'enzyme';
 import Title from '../components/Title';
 
 describe('Title component', () => {
+  const props = {
```

Comment:
Is this related to the ticket?

Reply:
@dkayiwa  in order to get the test coverage up and pass from coverall had to write some unit tests apart from the ticket after talking wih @abulojoshua1.

Reply:
Was the coverage drop caused by changes in this pull request?

Reply:
Nope u can check that in coveralls

Reply:
This pull request is about only one thing. And that is: Add mapping for existing concept.
Any other issues should have separate tickets and pull requests.

Reply:
@dkayiwa actuallly htis is only covering  Add mapping for existing concept.But becuase of the coverage drop @abulojoshua1  asked to add more unit tests

Reply:
If the tests are not covering what this ticket is all about, then they are in a wrong pull request.

Reply:
@dkayiwa tests covers all in the story.And as well as it tries to address coverage drop with new unit tests

Reply:
Just to give you one example, how is this test related to? "Add mapping for existing concept "

Reply:
@dkayiwa  done changes Daniel